docs/integrations/community-nodes/usage.md

```diff
@@ -0,0 +1,12 @@
+# Using community nodes
+
+To use community nodes, you first need to [install](/integrations/community-nodes/installation/){:target=_blank} them.
+
+## Adding community nodes to your workflow
+
+After installing a community node, you can use it like any other node. n8n displays the node in search results in the **Nodes** panel. To show which nodes are community nodes and which are built-in, n8n marks community nodes with a **Package** <span class="inline-image">![Package icon](/_images/common-icons/package.png)</span> icon.
```

Comment:
```suggestion
After installing a community node, you can use it like any other node. n8n displays the node in search results in the **Nodes** panel. n8n marks community nodes with a **Package** <span class="inline-image">![Package icon](/_images/common-icons/package.png)</span> icon.
```

Just a thought: When I read "which are X vs which are Y", I expect the following explanation to have the same structure, like "X are marked with #, Y are marked with *".  If I get only the first part, to me it sounds like something is missing and leaves me thinking "so Y are marked with something else, or are not marked at all?".